modules/tinymce/src/core/main/ts/keyboard/CefNavigation.ts

```diff
@@ -91,8 +92,25 @@ const moveToLineEndPoint = (editor: Editor, forward: boolean): boolean => {
   return NavigationUtils.moveToLineEndPoint(editor, forward, isCefPosition);
 };
 
+const selectToEndPoint = (editor: Editor, forward: boolean): boolean =>
+  getEdgeCefPosition(editor, !forward)
+    .map((pos) => {
+      const rng = pos.toRange();
+      const curRng = editor.selection.getRng();
+      forward ? rng.setStart(curRng.startContainer, curRng.startOffset) : rng.setEnd(curRng.endContainer, curRng.endOffset);
+      return rng;
+    })
+    .fold(
+      Fun.never,
+      (rng) => {
+        NavigationUtils.moveToRange(editor, rng);
+        return true;
+      }
+    );
```

Comment:
A `fold` where the none case is `Fun.never` is the same as using `exists` instead.
```suggestion
    .exists((rng) => {
      NavigationUtils.moveToRange(editor, rng);
      return true;
    });
```